app/src/org/commcare/activities/MenuList.java

```diff
@@ -67,6 +71,11 @@ public String getActivityTitle() {
         return null;
     }
 
+    @Override
+    public boolean isBackEnabled() {
+        return !(CommCareApplication._().isConsumerApp() && isRootModuleMenu);
+    }
```

Comment:
Let's plan on refactoring `MenuGrid` and `MenuList` at some point to subclass the same parent, so this, and other code, doesn't need to be duplicated